node/pkg/driver/node_utils.go

```diff
@@ -70,7 +70,7 @@ type NodeUtilsInterface interface {
 	ParseIscsiInitiators() (string, error)
 	GetInfoFromPublishContext(publishContext map[string]string, configYaml ConfigFile) (string, int, map[string][]string, error)
 	GetArrayInitiators(ipsByArrayInitiator map[string][]string) []string
-	GetSysDevicesFromMpath(baseDevice string) (string, error)
+	GetSysDevicesFromMpath(device string) ([]string, error)
```

Comment:
why not `baseDevice`?

Reply:
to align with [implementation](https://github.com/IBM/ibm-block-csi-driver/blob/99463e0043bf14f7fa9df35c65bbe8c40a668910/node/pkg/driver/node_utils.go#L157)

Reply:
I see.
IMO in this case, it would be better to align the implementation instead, since `device` is too general

Reply:
done